tests/data/jsons/all_ok.json

```diff
@@ -8,13 +8,13 @@
         "alertname": "DiskSpace",
         "device": "rootfs",
         "fstype": "rootfs",
-        "instance": "cs30.evilcorp",
```

Comment:
It's only an aesthetic thing, but I would either change L18 to `idealista.Test` as well (and all the instances and summaries in all the json files) or switch back to the previous (and geekier :P ) value. Other than that, really nice job ;)

Reply:
Switched back to the MR R reference :smile: